### ISSUE TITLE
H3 in Order History page jumps heading levels

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#13436]
**Describe the bug**
- Automated test detects heading jumping level.
- Heading is also not meaningful as it repeats the content of the H1.

**Tell us the version of Spartacus**
- Library version: 4.0.0

**To Reproduce**

1. Go to any product and submit an order.
2. Go to Order History
3. See error

**Expected behavior**
- This should be an H2
- Find a meaningful text for the heading to avoid being repetitive (check with Designers).

**Screenshots**
![Screen Shot 2021-08-05 at 7 32 09 PM](https://user-images.githubusercontent.com/15113219/128568109-6cf09dcc-c75a-4fd0-8479-8c79f48ce1a8.png)

**Comment by saad-mo** [comment]
change `h3` to `h2` in `order-hidstory.component.html`

breaking change

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

Order History is H2 now. And it reads the text "Order History"

![OrderHistoryH2](https://user-images.githubusercontent.com/45409515/155800408-6c500f92-b4c9-46da-9aea-adfca626f90c.png)

<img width="1216" alt="Screen Shot 2022-02-25 at 3 43 33 PM" src="https://user-images.githubusercontent.com/45409515/155800410-e0200996-5d7d-4bcf-a858-b89ed01707cd.png">
